templates/zerver/compose.html

```diff
@@ -100,7 +100,7 @@
                   <div id="send_controls" class="new-style">
                     <label id="enter-sends-label" class="compose_checkbox_label" for="enter_sends">{{ _('Press Enter to send') }}&nbsp;</label>
                     <input type="checkbox" id="enter_sends" name="enter_sends" value="enter_sends" />
-                    <button type="submit" id="compose-send-button" class="button small send_message" tabindex="150">{{ _('Send') }}</button>
+                    <button type="submit" id="compose-send-button" class="button small send_message" tabindex="0">{{ _('Send') }}</button>
```

Comment:
Does changing this tabindex actually fix a problem?  I vaguely recall that we intentionally set it super high, and testing, it appears that the high tab index means that hitting tab one too many times with compose open does nothing, rather than going to the users list in the right sidebar (which can be annoying for people using `tab+enter` to send messages, since then hitting `tab+tab+enter` means something totally random happens)